docs/docs/schema-customization.md

```diff
@@ -252,6 +258,9 @@ exports.createSchemaCustomization = ({ actions }) => {
 
 It is useful to think about your data, and the corresponding GraphQL schema, by
 always starting from the Node types created by source and transformer plugins.
+This is now helping us to create a new type called Frontmatter that can be further 
+used in mutation queries to be used and add this Frontmatter typed data to the database
+through our Graphql query.
```

Comment:
This feels out of place here and I'm not entirely sure the sentiment is accurate.